rotkehlchen/exchanges/iconomi.py

```diff
@@ -258,10 +259,37 @@ def query_balances(self, **kwargs: Any) -> ExchangeQueryBalances:
 
         for balance_info in resp_info['daaList']:
             ticker = balance_info['ticker']
-            self.msg_aggregator.add_warning(
-                f'Found unsupported ICONOMI strategy {ticker}. '
-                f' Ignoring its balance query.',
-            )
+
+            if ticker == 'AUSTS':
+                # The AUSTS strategy is 'ICONOMI Earn'. We know that this strategy holds its
+                # value in Anchor UST (AUST). That's why we report the user balance for this
+                # strategy as usd_value / AUST price.
+                try:
+                    aust_usd_price = Inquirer().find_usd_price(asset=A_AUST)
+                except RemoteError as e:
+                    self.msg_aggregator.add_error(
+                        f'Error processing ICONOMI balance entry due to inability to '
+                        f'query USD price: {str(e)}. Skipping balance entry',
+                    )
+                    continue
+
+                try:
+                    usd_value = deserialize_fval(balance_info['value'], 'usd_value', 'iconomi')
+                except (DeserializationError, KeyError) as e:
+                    self.msg_aggregator.add_warning(
+                        f'Skipping iconomi balance entry {balance_info} due to {str(e)}',
+                    )
+                    continue
+
+                assets_balance[A_AUST] = Balance(
+                    amount=usd_value / aust_usd_price,
```

Comment:
`find_usd_price()` can return `ZERO`, which would make this a division by zero. Since this logic probably wont' work with zero price bail out above if it is `ZERO`.